src/ui/SmartSnippet/UserFeedbackBanner.ts

```diff
@@ -74,17 +74,18 @@ export class UserFeedbackBanner {
 
   private buildContainer() {
     return $$(
-      'fieldset',
+      'div',
       {
-        className: CONTAINER_CLASSNAME
+        className: CONTAINER_CLASSNAME,
+        ariaLabelledby: this.labelId
```

Comment:
Documented [here](https://www.w3.org/WAI/GL/wiki/Using_grouping_roles_to_identify_related_form_controls#:~:text=The group can be labeled using aria-labelledby)